external-crates/move/crates/move-analyzer/tests/ide_testsuite.rs

```diff
@@ -244,6 +255,58 @@ impl CursorTest {
     }
 }
 
+impl HintTest {
+    fn test(
+        &self,
+        test_idx: usize,
+        symbols: &Symbols,
+        output: &mut dyn std::io::Write,
+        use_file_path: &Path,
+    ) -> anyhow::Result<()> {
+        let inlay_hints = inlay_hints_internal(
+            symbols,
+            use_file_path.to_path_buf(),
+            /* type_hints */ true,
+            /* param_hints */ true,
+        )
+        .unwrap();
+        let lsp_line = self.use_line - 1; // 0th-based
```

Comment:
Would it be easier to maintain these tests if both were 1-indexed or 0-indexed?

Reply:
Resolved in https://github.com/MystenLabs/sui/pull/18589 based on our discussion